frontend/src/components/Header/Header.tsx

```diff
@@ -0,0 +1,22 @@
+import './Header.css';
+
+import { ReactComponent as InstagramIcon } from 'assets/icons/instagram-icon.svg';
+import { ReactComponent as DownIcon } from 'assets/icons/down-icon.svg';
+import { Search, Navbar } from './../';
+
+interface HeaderProps {
+  onCreatePostClick: () => void;
+}
+
+export function Header({ onCreatePostClick }: HeaderProps) {
+  return (
+    <div className="header flex-box">
```

Comment:
Is `div` a semantically appropriate element to use for a component that is the _header_ of the app?